packages/checkbox/src/index.mts

```diff
@@ -106,13 +110,16 @@ export default createPrompt(
     const [showHelpTip, setShowHelpTip] = useState(true);
     const [errorMsg, setError] = useState<string | undefined>(undefined);
 
-    useKeypress((key) => {
+    useKeypress(async (key) => {
       if (isEnterKey(key)) {
+        const isValid = await validate(items);
```

Comment:
Did you consider sending in only checked items VS all items?

I'm not sure which one will make for the better API 🤔 I'm wondering what you think?

Reply:
Didn't think about it, but it might make sense to send only checked items. The user wants to check the validity of an answer, which in this case should be only checked items and nothing else. This way, the user doesn't need to think about what is and what isn't checked and will receive just the checked items.

Reply:
Yeah, I agree. Let's only send checked items to the validation function then. Thanks!